src/app/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -157,13 +156,18 @@ export const SelectMaxBidQueryRenderer: React.FC<{
           artworkID,
           saleID,
         }}
-        render={renderWithLoadProgress<SelectMaxBidQuery["response"]>((props) => (
-          <SelectMaxBidContainer
-            me={props.me!}
-            sale_artwork={props.artwork!.sale_artwork!}
-            navigator={navigator}
-          />
-        ))}
+        render={renderWithLoadProgress<SelectMaxBidQuery["response"]>((props) => {
+          if (props.artwork?.sale_artwork && props.me) {
+            return (
+              <SelectMaxBidContainer
+                me={props.me}
+                sale_artwork={props.artwork.sale_artwork}
+                navigator={navigator}
+              />
+            )
+          }
+          return null
```

Comment:
https://medium.com/swlh/return-early-pattern-3d18a41bba8 🤷 

```suggestion
          if (!props.artwork?.sale_artwork || !props.me) {
            return null
          }
            return (
              <SelectMaxBidContainer
                me={props.me}
                sale_artwork={props.artwork.sale_artwork}
                navigator={navigator}
              />
            )
```

Reply:
good note, I copied this code and didn't notice this